pytest.ini

```diff
@@ -92,7 +94,6 @@ addopts =
     -p no:logging
     --basetemp=/tmp/pytest
     --strict-markers
-    --tc-file=tests/global_config.py
```

Comment:
Don't we prefer to keep `global_config.py` as the default option, and only override it when a non-default arch is requested?